docs/source/guidelines.rst

```diff
@@ -156,55 +296,31 @@ properties. See an example below:
-
-
-Testing
-=======
-
-pytest framework is used for unit testing.
-The test modules are located in simtools/test (note the differences between unit and integration tests).
-Every module should have its respective test module and
-ideally all functions should be covered by tests.
-
-It is important to write the tests in parallel with the modules
-to assure that the code is testable.
+Docker Container for Development
+=================================
 
-The pytest decorators mark.ignoreif are used to mark the tests that
-requires: a) a config file properly set, b) a sim_telarray installation and
-c) DB connection. Each of these are identified before each pytest session
-and environment variables are used to store this information. See the implementation
-in conftest.py. In util/tests.py one can find functions that reads these variables.
+A docker container is made available for developers, see the `Gammasim-tools container repository <https://github.com/gammasim/containers/tree/main/dev>`_.
+The container has the python packages, CORSIKA, and sim_telarray pre-installed.
+Setting up a system to run gammasim-tools applications or tests should be a matter of minutes:
 
+\1. install Docker and start the Docker application (see `Docker installation page <https://docs.docker.com/engine/install/>`_). Other container systems like Apptainer, Singularity, Buildah/Podman, etc should work, but are not thoroughly tested.
 
-Integration with CORSIKA and sim_telarray
-=========================================
+2. obtain the access parameters for the CTA Simulation Model data base and write a small script ``set_DB_environ.sh`` to set these parameters to be used in the container:
+.. code-block::
+    export DB_API_USER=<db_user_name>
+    export DB_API_PW=<db_password>
+    export DB_API_PORT=<db_port>
+    export DB_SERVER=<db_server>
 
-CORSIKA and sim_telarray are external tools to simtools. Their integration should be
-minimally coupled with the rest of the package. The modules that depend directly on these
-tools should be connected to the rest of the package through interfaces. This way, it
-will be easier to replace these tools in the future.
+3. Start up a container and e.g. run the Gammasim-tools unit tests using the following commands:
+.. code-block::
 
-One example of this approach is shower_simulator module, that is an interface module that
-connects to the tool specific module that is meant to manage shower simulations. In this case,
-this tool specific module is the corsika_runner, which can be replaced in the future.
+    # create a working directory
+    mkdir external && cd external
+    # clone gammasim-tools repository
+    git clone https://github.com/gammasim/gammasim-tools.git
+    # startup a container (download if is not available in your environment)
+    docker run --rm -it -v "$(pwd)/external:/workdir/external" ghcr.io/gammasim/containers/gammasim-tools-dev:v0.3.0-dev1 bash -c "$(cat ./entrypoint.sh) && bash"
```

Comment:
Is `entrypoint.sh` provided with gammasim-tools or should it be given here in the doc as well?